libtransmission/clients.c

```diff
@@ -279,10 +279,6 @@ char* tr_clientForId(char* buf, size_t buflen, void const* id_in)
         {
             four_digits(buf, buflen, "Avicora", id + 3);
         }
-        else if (strncmp(chid + 1, "BB", 2) == 0)
```

Comment:
The other condition down below needs to be removed instead to avoid the change in behavior.